src/common/datetime/localize_date.ts

```diff
@@ -1,46 +1,30 @@
 import memoizeOne from "memoize-one";
 
-// Sunday = 0
-const localizeWeekday = (
-  language: string,
-  day_of_week: number,
-  short: boolean
-): string => {
-  const date = new Date(Date.UTC(1970, 0, 1 + 3 + day_of_week));
-  return new Intl.DateTimeFormat(language, {
-    weekday: short ? "short" : "long",
-    timeZone: "UTC",
-  }).format(date);
-};
-
 export const localizeWeekdays = memoizeOne(
```

Comment:
Last question, how often do we call this function, is it need to be memoized?

Reply:
I don't pretend to understand the full page load sequence, but just by breakpoints and log messages looks like the Vue date-range-picker createElement function is called 4 times every time we navigate to the energy page (+1 more time when the picker is expanded), and 5 times every time we navigate to the history page (+1 more time when the picker is expanded).

Reply:
Also seems to be called 6 more times every time we change an entity in the history panel 😕

Reply:
Seems less than ideal, but another issue 👍